docs/metrics.md

```diff
@@ -114,3 +115,28 @@ All connection pool metrics have the following tags:
 | ----------- | --------- |
 | `pool.name` | The name of the connection pool: Spring bean name if Spring is used, the JMX object name otherwise.
 | `pool.type` | The type/implementation of the connection pool: e.g. `c3p0`, `dbcp2`, `hikari`, `tomcat-jdbc`, `vibur-dbcp`.
+
+### Thread pool metrics
+
+Splunk Distribution of OpenTelemetry Java instruments several thread pool implementations:
+
+* [Tomcat connector thread pools](https://tomcat.apache.org/tomcat-8.5-doc/index.html)
+
+Each of the supported connection pools reports a subset of the following metrics:
+
+| Metric name                | Description |
+| -------------------------- | ----------- |
+| `executor.threads`         | The current number of threads in the pool.
+| `executor.threads.active`  | The number of threads that are currently busy.
+| `executor.threads.idle`    | The number of threads that are currently idle.
+| `executor.threads.core`    | Core thread pool size - the number of threads that are always kept in the pool.
+| `executor.threads.max`     | The maximum number of threads in the pool.
```

Comment:
I definitely understand the need to have `core` and `max` accounted for, but they really don't change during the lifespan of a pool, right? It's an interesting idea to report the same metric repeatedly.....hmmm....not sure what to do about it, just observing.

Reply:
You actually can change those by calling `setCorePoolSize()`/`setMaximumPoolSize()` - I don't think that anybody does that though, they're effectively final.